packages/client/lib/service/txpool.ts

```diff
@@ -283,27 +282,27 @@ export class TxPool {
         this.validateTxGasBump(existingTxn.tx, tx)
       }
     }
-    const block = await this.service.chain.getLatestHeader()
-    if (block.baseFeePerGas) {
-      if (currentGasPrice.maxFee.lt(block.baseFeePerGas)) {
-        throw new Error(
-          `Tx cannot pay basefee of ${block.baseFeePerGas}, have ${currentGasPrice.maxFee}.`
-        )
-      }
+    const latest = this.service.chain.headers.latest!
+    if (latest.baseFeePerGas && maxFee.lt(latest.baseFeePerGas.divn(2)) && !isLocalTransaction) {
```

Comment:
I have matched Geth's logic here, Geth rejects txs which cannot pay base fee. I think this is OK though :+1:

Reply:
ah ok, thanks yeah I just wanted some flexiblity, if we are in a temporary time of very high basefee it still makes sense to accept at a lower fee because it can eventually go back down, and strictly validating at the current basefee would probably miss a good amount of txs? perhaps geth does it to avoid a DDOS vector where during high basefee people flood with low priced txs which can drown out the higher fee txs.